### PR TITLE
Updated Open Data section to add and fill the QF columns based on the…

### PR DESCRIPTION
**Pull request by amyethomson**
… previous submissions

### CODE REVIEW

File: Run publication.R [hunk 1]
```diff
@@ -6,6 +6,7 @@ library('xlsx')
 library('lubridate')
 library('stringr')
 library("RcppBDT")
+library('zoo')
```
Comment (ksimonova) [inline Run publication.R]: Substitute with fuction that installs the packages if needed. Substitute xlsx with readxl

rm(list=ls()) # Ensuring the environemtn is clean

#### Loading required libraries (will install if required) ----
required_libraries <- c('tidyverse','stringr','zoo','RcppBDT','readxl','lubridate','reshape2')

for (i in required_libraries){
  if( !is.element(i, .packages(all.available = TRUE)) ) {
    install.packages(i)
  }
  library(i,character.only = TRUE)
}

Comment (ksimonova) [inline Run publication.R]: Substitute with fuction that installs the packages if needed. Substitute xlsx with readxl

rm(list=ls()) # Ensuring the environemtn is clean

#### Loading required libraries (will install if required) ----
required_libraries <- c('tidyverse','stringr','zoo','RcppBDT','readxl','lubridate','reshape2')

for (i in required_libraries){
  if( !is.element(i, .packages(all.available = TRUE)) ) {
    install.packages(i)
  }
  library(i,character.only = TRUE)
}

File: Run publication.R [hunk 2]
```diff
@@ -511,13 +512,91 @@ board <- open %>%
...
 #Create hospital level file
 hospital <- open %>%
               select(-HBT2014)
       
-#Output files
+
+###Compare new files to last month's files
+
+#Calculate dates for last month's file name
+monthcom <- month(Sys.Date() %m-% months(2))
```
Comment (ksimonova) [inline Run publication.R]: All ran fine.

File: Run publication.R (hunk 1 above)
Comment (amyethomson) [reply Run publication.R]: Thanks, Kristin. We used to have rm(list=ls()) in the scripts but I have removed this as it's not good practice to include it especially if you're sharing your code.

**Pull request merged**